src/Input/Checkbox/Checkbox.test.tsx

```diff
@@ -69,6 +82,16 @@ describe('when it is rendered', () => {
     expect(checkboxInput.checked).toEqual(false);
   });
 
+  it('should display a checked checkbox if checked=true', () => {
+    const { checkboxInput } = setupCheckbox({ ...props, checked: true });
+    expect(checkboxInput.checked).toEqual(true);
+  });
+
+  it('should display an unchecked checkbox if checked=false', () => {
+    const { checkboxInput } = setupCheckbox({ ...props, checked: false });
+    expect(checkboxInput.checked).toEqual(false);
+  });
+
```

Comment:
I think we can consider checking controlled value with rerender. 

like this
```
describe('<Select/> handleValueChange', () => {
  it('input value should be the same as passed prop value', async () => {
    const Component = ({ value }: { value: string }) => (
      <Select label="handleValueChange" value={value}>
        <Select.Option value="handleValueChange">
          handleValueChange
        </Select.Option>
      </Select>
    );

    const mockValue = 'mock';
    const { queryByRole, rerender } = render(<Component value="" />);
    const selectInput = queryByRole('combobox');
    expect(selectInput.getAttribute('value')).toEqual('');
    rerender(<Component value={mockValue} />);
    expect(selectInput.getAttribute('value')).toEqual(mockValue);
    rerender(<Component value="" />);
    expect(selectInput.getAttribute('value')).toEqual('');
  });
});
```

Reply:
Hmm but value isn't the property that's being controlled..? `checked` is the one that we need to be concerned about. Not quite sure what's the use-case for changing/controlling `value`.

Reply:
Oh sorry for being unclear, yeah I meant `checked` not `value` haha. 
So it might look like this. 
```
    const Component = props => <Checkbox {...props} />;

    const { rerender, getByLabelText } = render(
      <Component {...props} checked={false} />
    );

    const checkboxInput = getByLabelText(
      props.value as string
    ) as HTMLInputElement;
    expect(checkboxInput.checked).toEqual(false);

    rerender(<Component checked={true} />);

    expect(checkboxInput.checked).toEqual(true);

```

Reply:
Oh yeah that makes sense. I've added the test case above.